frontends/pytorch/test/lit.site.cfg.py.in

```diff
@@ -50,4 +51,4 @@ import lit.llvm
 lit.llvm.initialize(lit_config, config)
 
 # Let the main config do the real work.
-lit_config.load_config(config, "@CMAKE_SOURCE_DIR@/frontends/pytorch/test/lit.cfg.py")
+lit_config.load_config(config, "@CMAKE_CURRENT_SOURCE_DIR@/lit.cfg.py")
```

Comment:
I think this change is a safe way to make this path agnostic to CMAKE_SOURCE_DIR, which can be different for an external build. It works on my computer and the GitHub check, but please correct me if I'm wrong.

Reply:
I'm ok with it if it works. Can't think of anything better without fiddling with it.